puma/histogram.py

```diff
@@ -34,12 +36,21 @@ def __init__(
         Parameters
         ----------
         values : np.ndarray
-            Input data for the histogram
+            Input data for the histogram. If bin_edges is specified (not None)
+            then this array is treated as the bin heights.
         weights : np.ndarray, optional
             Weights for the input data. Has to be an array of same length as the input
             data with a weight for each entry. If not specified, weight 1 will be given
             to each entry. The uncertainties are calculated as the square root of the
             squared weights (for each bin separately). By default None.
+        bin_edges : np.ndarray, optional
+            If specified, the histogram is considered "filled": the array given to
+            values is treated as if it was the bin heights corresponding to these
+            bin_edges and the "weights" input is ignored. By default None.
+        sum_squared_weights : np.ndarray, optional
+            Only considered if the histogram is considered filled (i.e bin_edges
```

Comment:
In these cases it's good to verify in the constructor that `sum_squared_weights` is `None` if `bin_edges` is not defined, this can prevent someone getting mixed up

Reply:
I have added a warning being logged for when this case happens